examples/firmware/esp32_basic/main.cpp

```diff
@@ -0,0 +1,16 @@
+#include "RESTuino/restuino_func.h"
+
+uint8_t len_ssid = 2;
+const char *ssid_def[] = {"ssid1", "ssid2"};
+const char *ssid_pass[] = {"pass1", "pass2"};
+
+static RestuinoFunc func;
+
+void setup()
+{
+  func.restuino_setup();
+}
+void loop()
+{
+  func.restuino_loop();
```

Comment:
One problem in making it into a library is that only `func.restuino_loop()` can be written in a loop function, so it must be written somewhere.